### PR TITLE
Updating 02-numpy numpy.diff example to use patient data

### PR DESCRIPTION
**Pull request by noatgnu**
This is a pull request for issue https://github.com/swcarpentry/python-novice-inflammation/issues/780

Change the lesson example from using random data to using patient 3 week 1 data from the included inflammation dataset. Variable name has also been changed to be more descriptive of the data it contains.

Question in the 2D array `numpy.diff` example added introduction to `axis` argument.

### CODE REVIEW

File: _episodes/02-numpy.md [hunk 1]
```diff
@@ -665,38 +665,38 @@ which is the average inflammation per patient across all days.
...
-> between two successive values. First we consider a one-dimensional
-> array of length 5. This could be part of some row `i` of our inflammation data,
-> i.e. `row_start = data[i,:5]`.
->
+> between two successive values. Let's use it to examine the changes
+> each day across the first week of patient 3 from our inflammation dataset.
+> 
 > ~~~
-> row_start = numpy.array([ 0,  2,  5,  9, 14])
+> patient3_week1 = data[3, :7]
```
Comment (ldko) [inline _episodes/02-numpy.md]: Looking at this in the rendered lesson now, I am thinking we should add a `print(patient3_week1)` to this Python block and add an output block:
```
>
> ~~~
>  [0. 0. 2. 0. 4. 2. 2.]
> ~~~
> {: .output}
```



This way, people can see the connection between the numbers in the array and the calculations shown below.

File: _episodes/02-numpy.md [hunk 2]
```diff
@@ -665,38 +665,38 @@ which is the average inflammation per patient across all days.
...
 >
 > ~~~
-> numpy.diff(row_start)
+> numpy.diff(patient3_week1)
 > ~~~
 > {: .language-python}
 >
 > ~~~
-> array([2, 3, 4, 5])
+> array([ 0,  2, -2,  4, -2,  0])
```
Comment (ldko) [inline _episodes/02-numpy.md]: When I run the code, I get the output with dots like:

`array([ 0.,  2., -2.,  4., -2.,  0.])`

so perhaps we should include the dots to match what people will actually see. (I know I am running an older version of Python 3 and probably `numpy` though, so if the output you already have here is what you get in actuality, please disregard this comment).

**Comment by maxim-belkin** [comment]
> Thanks, Toan! I have a couple comments about possible changes, and perhaps @maxim-belkin also wants to provide feedback.

Thanks, Lauren. I second both of your comments.

Toan: Thank you for working on this. We'll merge this PR once you make the changes suggested by Lauren.

Note that **after** we merge this PR, you will have to synchronize your fork with our repository because you used `gh-pages` branch for the pull request. Specifically, you will have to execute the following set of comments on the command line in the directory containing this lesson on your computer:

```sh
git fetch https://github.com/swcarpentry/python-novice-inflammation
git reset --hard FETCH_HEAD
git push --force https://github.com/noatgnu/python-novice-inflammation
```
Again, thank you for your contribution and please let us know if you'd like to work on something else.

Maxim

**Comment by maxim-belkin** [comment]
A gentle ping @noatgnu

**Comment by ldko** [comment]
Awesome! Thanks for making the changes @noatgnu .

**Pull request merged**